Makefile

```diff
@@ -1,7 +1,7 @@
 CC=gcc
-CFLAGS=--std=c99 -g -O2 -Wall --pedantic `freetype-config --cflags` `sdl-config --cflags`
-LDFLAGS=`icu-config --ldflags`
-LIBS=-lcairo -lharfbuzz -lharfbuzz-icu `freetype-config --libs` `sdl-config --libs`
+CFLAGS=--std=c99 -g -O2 -Wall --pedantic `pkg-config --cflags` `sdl2-config --cflags`
+#LDFLAGS=`icu-config --ldflags`
+LIBS=-lcairo -lharfbuzz -lharfbuzz-icu `pkg-config --cflags --libs freetype2` `pkg-config --libs` `sdl2-config --libs`
```

Comment:
Should remove the `--cflags` here, and remove the extra `pkg-config --libs`

Reply:
yeah this `pkg-config --libs` : does nothing 😅